third_party/dlpack/dlpack.h

```diff
@@ -0,0 +1,173 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file dlpack.h
```

Comment:
no license?

Reply:
Maybe we can dlpack as a subrepo, so license will be there?

Reply:
Done